src/primevue/toast/toast.stories.ts

```diff
@@ -0,0 +1,139 @@
+import { Meta, StoryObj } from "@storybook/vue3";
+import Toast from "primevue/toast";
+import { useToast } from "primevue/usetoast";
+import { html } from "@/lib/tags.ts";
+import Btn from "primevue/button";
+import IconCheck from "~icons/ic/baseline-check-circle-outline";
+
+
+const meta: Meta<typeof Toast> = {
+    component: Toast,
+    tags: ["autodocs"],
+    args: {
+        position: 'top-right',
+    },
+
+    argTypes: {
+        position: {
+            control: { type: 'select' },
+            options: [
+                'top-right', 'top-left',
+                'bottom-right', 'bottom-left',
+            ],
+        },
+    },
+};
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Success: Story = {
+    render: (args) => ({
+        components: { Toast, Btn, IconCheck },
+        setup() {
+            const toast = useToast();
+
+            const showSuccess = () => {
+                toast.add({
+                    severity: 'success',
+                    summary: 'Verkündung erfolgreich hochgeladen',
+                    detail: 'Sie können mit der Arbeit an der neuen Verkündung beginnen.',
+                    life: 3000
```

Comment:
Suggestion: can you remove these `life` params? It makes it harder to properly review as the toasts just disappear all the time 😁 if you want to demonstrate it, you can add a separate story that includes `life`.

Reply:
Todo: On a related note though, pressing "close" does not make the toast disappear. Perhaps there is some Storybook setup needed, or some additional styling.